src/components/OptimismDowntimeWarning/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { Trans } from '@lingui/macro'
+import { SupportedChainId } from 'constants/chains'
+import { useActiveWeb3React } from 'hooks/web3'
+import { AlertOctagon } from 'react-feather'
+import styled from 'styled-components/macro'
+
+const Root = styled.div`
+  background-color: ${({ theme }) => theme.yellow3};
+  border-radius: 18px;
+  color: black;
+  margin-top: 16px;
+  max-width: 480px;
+  padding: 16px;
+  width: 100%;
+`
+const WarningIcon = styled(AlertOctagon)`
+  margin: 0 8px 0 0;
+`
+const TitleRow = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  margin: 0;
+  font-size: 20px;
+  font-weight: 600;
+  line-height: 25px;
+`
+const Body = styled.div`
+  display: inline-block;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 8px 0 0 0;
+`
+const LinkOutToNotion = styled.a`
+  color: black;
+`
+
+export default function OptimismDowntimeWarning() {
+  const { chainId } = useActiveWeb3React()
+  if (!chainId || ![SupportedChainId.OPTIMISM, SupportedChainId.OPTIMISTIC_KOVAN].includes(chainId)) {
+    return null
+  }
+  return (
+    <Root>
+      <TitleRow>
+        <WarningIcon />
+        Optimism <Trans>Scheduled Downtimes</Trans>
```

Comment:
Might be better to put 'Optimism' inside the `<Trans>` (thanks for remembering to wrap the new strings inside `<Trans>` btw). Some languages have word different word ordering

Reply:
Oh interesting point... Do you know how our translation setup would work with a word like Optimism that's being used as a proper noun here?

Reply:
I think the easiest way to handle this would be to put optimism in a variable

`const name = 'Optimism'`

`<Trans>{name} Scheduled Downtimes`

such that the translation key is something like `{0} Scheduled Downtimes` and translator don't attempt to translate the word `Optimism`.

Reply:
that makes sense
also, yuck